<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="switchLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Switches</string>
    </property>
   </widget>
   <widget class="QLabel" name="SwitchSec">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Section</string>
    </property>
   </widget>
   <widget class="QLabel" name="SwitchBlock">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Block</string>
    </property>
   </widget>
   <widget class="QLabel" name="SwitchConn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="Line" name="divider1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="RRXing">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Railroad Crossings</string>
    </property>
   </widget>
   <widget class="QLabel" name="RRRedLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red Line</string>
    </property>
   </widget>
   <widget class="QLabel" name="RRGreenLine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green Line</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>E-19</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>I-47</string>
    </property>
   </widget>
   <widget class="Line" name="divider2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="OccHeader">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Occupancy</string>
    </property>
   </widget>
   <widget class="QLabel" name="OccSec">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Section</string>
    </property>
   </widget>
   <widget class="QLabel" name="OccBlock">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Block</string>
    </property>
   </widget>
   <widget class="QLabel" name="SetTo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="OccLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="Line" name="divider3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="divider4">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>380</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="TrackHeaters">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Track Heaters</string>
    </property>
   </widget>
   <widget class="QLabel" name="InduceFault">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Induce Fault</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeaterStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heater Status: ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date/Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setOccupied">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Occupied</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setVacant">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Vacant</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="SwitchSec_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="EnterTemp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Temp</string>
    </property>
   </widget>
   <widget class="QComboBox" name="OccLineSel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="OccSecSel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="OccBlockSel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Connection">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
